src/BUILD.gn

```
@@ -14,24 +15,29 @@ target("shared_library", "v8jsi") {
     "jsi/jsilib-windows.cpp",
     "jsi/jsilib.h",
     "jsi/threadsafe.h",
-    "napi/env-inl.h",
-    "napi/js_native_api_v8_internals.h",
-    "napi/js_native_api_v8.cc",
-    "napi/js_native_api_v8.h",
-    "napi/js_native_ext_api_v8.cpp",
-    "napi/util-inl.h",
-    "public/compat.h",
-    "public/js_native_api_types.h",
-    "public/js_native_api.h",
-    "public/js_native_ext_api.h",
-    "public/NapiJsiRuntime.cpp",
     "public/ScriptStore.h",
     "public/V8JsiRuntime.h",
     "V8JsiRuntime_impl.h",
     "V8JsiRuntime.cpp",
     "IsolateData.h",
   ]
 
+  if (v8jsi_enable_napi) {
```

should we use the full name?
```suggestion
  if (v8jsi_enable_node_api) {
```